client/src/Pages/Login.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from "react";
+import Navbar from "../Components/Navbar";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import i18 from 'i18next'
+import { Link } from "react-router-dom";
+
+const Login = () =>{
+
+    const [values,setValues] = useState({
+        gmail: '',
+        password: ''
```

Comment:
login is only possible with `gmail` add or any other email address also possible?

Reply:
For now it is not in use the `Login.jsx` and `Signup.jsx` page isn't in use.
As we are using `auth0` as user management system.